consensus/types/src/blob_sidecar.rs

```diff
@@ -34,15 +35,20 @@ pub struct BlobIdentifier {
 pub struct BlobSidecar<T: EthSpec> {
     pub block_root: Hash256,
     // TODO: fix the type, should fit in u8 as well
+    #[serde(with = "eth2_serde_utils::quoted_u64")]
     pub index: u64,
     pub slot: Slot,
     pub block_parent_root: Hash256,
+    #[serde(with = "eth2_serde_utils::quoted_u64")]
     pub proposer_index: u64,
+    #[serde(with = "ssz_types::serde_utils::hex_fixed_vec")]
     pub blob: Blob<T>,
     pub kzg_commitment: KzgCommitment,
     pub kzg_proof: KzgProof,
 }
 
+pub type BlobSidecars<T> = VariableList<BlobSidecar<T>, <T as EthSpec>::MaxBlobsPerBlock>;
```

Comment:
Since this is no longer defined by the spec can we opt for a more easy to differentiate name like `BlobSidecarList` (or anything more than a typo away from an existing type) ?

Reply:
is this actually a variable list? wouldn't it be a fixed size list?

Reply:
💯 💯